packages/scripts/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Breaking Changes
 
 - The bundled `puppeteer` (`^2.0.0`) dependency has been replaced with `puppeteer-core` in version `3.0.0`. Puppeteer uses Chromium v81 instead of Chromium v79. See the [full list of changes](https://github.com/puppeteer/puppeteer/releases/tag/v3.0.0). It also allowed preventing Chromium installation together with `@wordpress/scripts`. It happens now on-demand when running `test-e2e` script, and it re-triggers only when a new version is required.
+- Bundle analysis now runs with module concatenation disabled. This represents the size of individual modules more accurately, at the cost of not providing an exact byte-for-byte match to the final size in the production chunk.
```

Comment:
I think you merged this change without rebasing. It ended up in the section that was already published to npm:

https://github.com/WordPress/gutenberg/blob/master/packages/scripts/CHANGELOG.md#900-2020-04-30

Reply:
Oops, I did do that, yes. Whether it's a breaking change or not depends on interpretation; if we consider the bundle analysis to be part of the `scripts` package "user contract", then this is a breaking change, otherwise it's not.

Shall I prepare a PR moving this to the right section and making it a non-breaking change?

Reply:
Yes, it would be great 👍